PubnativeLite/PubnativeLite/Ad Model/HyBidAd.m

```diff
@@ -83,6 +83,16 @@ - (NSString *)htmlData {
     if (data) {
         result = data.html;
     }
+    
+//    NSString *path = [[NSBundle mainBundle] pathForResource:@"ad8" ofType:@"txt"];
+//    NSError *error;
+//    NSString *content = [NSString stringWithContentsOfFile:path encoding:NSUTF8StringEncoding error:&error];
+//
+//    if (error)
+//        NSLog(@"Error reading file: %@", error.localizedDescription);
+//
+//    NSLog(@"contents: %@", content);
+//    return content
```

Comment:
Are those commented out lines necessary? If not, let's remove them 👍

Reply:
Those are commented on purpose for test. I left also the test creative folder.